test/Tests/Adapters/TypeMappingTests.cs

```diff
@@ -34,7 +34,7 @@ public void Can_Derive_Nullable_Primitive_Types()
             Type<TimeSpan?>().ShouldBeOfType<TimeSpanGraphType>();
             Type<Id?>().ShouldBeOfType<Extended.IdGraphType>();
             Type<Url>().ShouldBeOfType<UrlGraphType>();
-            Type<Uri>().ShouldBeOfType<UriGraphType>();
+            Type<Uri>().ShouldBeOfType<ConventionsTypes.UriGraphType>();
```

Comment:
Can you put a `using UriGraphType = GraphQL.Conventions.Adapters.Types.UriGraphType;` on top instead of changing this in multiple places? Currently, you're picking the wrong type.